### DISCUSSION TITLE
Can not link a Bitbucket repo without creating a team and pay for pro plan

### DISCUSSION THREAD

**Discussion by NazeehAlhosary** [vercel/community#6563] [Help]
### Summary

I',m trying to add one of the projects/repo that I have on Bitbucket but I can not import the repos if I don't pay for the upgraded plan. I used to do that without any issues, could you please help me? (I am the admin of all the projects/repos)
The repos that under my name (default project in bitbucket) can be imported without any issue!

### Steps to Reproduce

One of the projects that is under my name (default project) directly (working as expected):
![image](https://github.com/vercel/community/assets/33482142/718ed432-f4e3-4941-8c5b-2138957292ab)

The project that I want to deploy (I have to pay):
![image](https://github.com/vercel/community/assets/33482142/8bf69e53-f9b8-428a-b8e3-6fb3228c0943)

**Comment by amyegan** [comment]
Hey @NazeehAlhosary. Is this a private repo under a workspace rather than under your name? That would explain why it works for other repos but not this one. A hobby account cannot be connected to a private repo in a git org workspace.

**Reply by NazeehAlhosary** [reply]
Hey @amyegan,
Thanks for your answer. First of all, you're right I'm using a private repository, but all of my existing projects in Vercel are related to the same project and are private. Even a few days back, I could add the same project, but I deleted it due to a reason, but when I tried to add it once again I couldn't.
I'm getting this error "You are attempting to import a Git repository from a GitHub Organization. Start a Pro Trial or upgrade to Pro." while I'm using Bitbucket not GitHub at all! check this ticket as well #6566

**Reply by NazeehAlhosary** [reply]
@amyegan  One more thing, I made the Project and the repository public for a few minutes, but still getting the same error as well!
